docs: Add global settings doc for admin > config (#8660)
sourcegraph/sourcegraph: doc/admin/config/global_settings.md

> # Global settings

Reviewer: Should this just be "Settings" and then describe the different levels/the cascade below?

Author: I wasn't sure about this, but I think changing to just Settings is right.

> Settings provide the ability to customize and control the Sourcegraph UI and user specific features. They do not configure operational aspects of the instance.

Reviewer: ```suggestion
Settings provide the ability to customize and control the Sourcegraph UI and user-specific features. They do not configure operational aspects of the instance (which are set in the [site configuration](site_config.md)).
```

> Settings are also different to [site configuration](site_config.md) as they are set the global level by site admins, the organization level by organization members, and at the individual user level.

Reviewer: ```suggestion
Settings can be set at the global level (by site admins), the organization level (by organization members), and at the individual user level.
```
